skins/cat17/templates/partials/membership/state_bar_sections.html.twig

```diff
@@ -1,6 +1,6 @@
 <li class="member-type">
 	<a href="#membership-type">
-		<span class="text" data-empty-text="{$ 'membership_section_membership_type_anchor' | trans | e( 'html_attr' ) $}"></span>
+		<span class="text" data-empty-text="{$ membership_type_empty_text | default( 'membership_section_membership_type_anchor' ) | trans | e( 'html_attr' ) $}"></span>
```

Comment:
Only if you really share my opinion on second thought... I am not a huge fan of defaults (i.e. optional parameters) in templates, in this case the `membership_type_empty_text`'s name does not even indicate it's a translation key (sounds more like an already translated message), is run through trans again.